crates/re_space_view/src/results_ext.rs

```diff
@@ -99,12 +105,19 @@ impl<'a> HybridLatestAtResults<'a> {
                 self.results
                     .get(component_name)
                     .and_then(|r| r.try_instance::<T>(&self.resolver, index)))
-            .or_else(|| {
-                // No override & no store -> try default instead
-                self.defaults
-                    .get(T::name())
-                    .and_then(|r| r.try_instance::<T>(&self.resolver, 0))
-            })
+    }
+
+    /// Utility for retrieving a single instance of a component.
+    ///
+    /// If overrides or defaults are present, they will only be used respectively if they have a component at the specified index.
```

Comment:
This text -- here and everywhere else -- is misleading: defaults don't care about index they just pick `0` always.
```suggestion
    /// If overrides are present, they will only be used respectively if they have a component at the specified index.